elastic/kibana
# [Infra, Profiling] Add KQL bar to Profiling tab in Infra

Closes https://github.com/elastic/kibana/issues/172690

## Summary

This PR Integrates `<EmbeddableProfilingSearchBar>` into the Profiling tab in Infra host details.

* Switched Profiling tab to use the date picker provided by `<EmbeddableProfilingSearchBar>` but made sure to keep it in sync with the date picker on other tabs
* Changed the profiling BE endpoints to accept the full `kuery` instead of just `hostname`

https://github.com/elastic/kibana/assets/793851/8b71a457-baa9-4ee7-ac8e-202de1b63753

## How To Test

1. Checkout locally and [setup profiling](https://github.com/elastic/observability-dev/blob/main/docs/profiling/on_prem.md)
2. Go to host details that has profiling data and make sure the KQL bar is present in "Universal Profiling" tab
3. Make sure the date picker changes stay consistent between Profiling tab and other tabs
4. Make sure the KQL bar filters the data while keeping it scoped to a single host
